src/repl.rs

```diff
@@ -116,40 +126,68 @@ impl Validator for PartiqlHelper {
             source = &source[4..];
         }
 
+        let source_len = source.len();
+        match source_len {
+            0 => return Ok(ValidationResult::Valid(None)),
+            _ => match &source[source_len - 1..source_len] {
+                ";" => {
+                    // TODO: there's a bug here where hitting enter from the middle of a query
```

Comment:
Can we also have an issue for this and mark it here?

Reply:
Sure, created https://github.com/partiql/partiql-rust-cli/issues/10 to track.